hadoop-hdds/rocksdb-checkpoint-differ/src/main/java/org/apache/ozone/rocksdiff/RocksDBCheckpointDiffer.java

```diff
@@ -711,24 +703,28 @@ public synchronized List<String> getSSTDiffList(
   /**
    * Core getSSTDiffList logic.
```

Comment:
```suggestion
   * Core getSSTDiffList logic.
   * For each sst in the src snapshot, traverse the dag to find its final
   * successor.  If any of those successors match an sst in the dest
   * snapshot, add it to the sameFiles map; (it doesn't need further
   * diffing).  Otherwise, add it to the differentFiles map, as it will
   * need further diffing.
```